test/crypto/SubtleCryptoProvider.spec.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const SubtleCryptoProvider = require('../../lib/crypto/SubtleCryptoProvider');
+const webcrypto = require('crypto').webcrypto;
+const expect = require('chai').expect;
+
+// webcrypto is only available on Node 15+.
+if (!webcrypto || !webcrypto.subtle) {
+  console.log(
+    `Skipping SubtleCryptoProvider tests. No 'webcrypto.subtle' available for ${process.version}.`
+  );
+  return;
+}
+
+const {createCryptoProviderTestSuite} = require('./helpers');
+
+describe('SubtleCryptoProvider', () => {
+  const provider = new SubtleCryptoProvider(webcrypto.subtle);
+
+  createCryptoProviderTestSuite(provider, true);
```

Comment:
nit: I try to avoid calling functions that call `describe` and `it` since this interacts poorly with test runners, but nbd